kb/pattern/scs/class.scs

```diff
@@ -0,0 +1,13 @@
+concept_your_class_identifier
+<- sc_node_class;
+<- class;
```

Comment:
```suggestion
```
There is no `class` in gwf pattern and I think it is not necessarily to add it. Remove in scs please